package.json

```diff
@@ -20,21 +20,22 @@
   },
   "dependencies": {
     "@justeat/f-dom": "^0.4.0",
-    "@justeat/f-footer": "^0.18.2",
-    "@justeat/f-header": "^0.26.0",
+    "@justeat/f-footer": "^0.21.0",
+    "@justeat/f-header": "^0.27.0",
     "@justeat/f-icons": "^0.8.0",
     "@justeat/f-toggle": "^0.13.0",
     "@justeat/f-validate": "^0.18.0",
-    "@justeat/fozzie": "^0.42.2",
+    "@justeat/fozzie": "^0.44.0",
+    "gift": "^0.10.2",
```

Comment:
What does this package do?

Reply:
Looks like it's some sort of cli tool for git but I don't see it being used it here.

Reply:
Is it worth removing if it's not in use?